climpred/metrics.py

```diff
@@ -195,24 +320,61 @@ def _less(forecast, reference, dim='svd', comparison=None):
           Prediction System.” Meteorologische Zeitschrift, December 21, 2016,
           631–43. https://doi.org/10/f9jrhw.
 
+    # TODO: not yet fully understood!
+
     Range:
         * pos: under-disperive
         * neg: over-disperive
         * perfect: 0
     """
-    if comparison.__name__ != '_m2r':
-        raise KeyError(
-            'LESS requires member dimension and therefore '
-            "compute_hindcast(comparison='m2r')"
-        )
-    numerator = _mse(forecast, reference, dim='member').mean(dim)
+    dim2 = 'init'  # [i for i in forecast.dims if i != 'member']
+    if 'init' in forecast.coords:
+        dim2 = 'init'
+    elif 'time' in forecast.coords:
+        dim2 = 'time'
+
+    forecast, ref2 = xr.broadcast(forecast, reference)
+    numerator = _mse(forecast, ref2, dim='member').mean(dim2)
     # not corrected for conditional bias yet
-    denominator = _mse(forecast.mean('member'), reference.mean('member'), dim=dim)
+    denominator = _mse(forecast.mean('member'), ref2.mean('member'), dim=dim2)
     less = np.log(numerator / denominator)
     return less
 
 
-def _bias(forecast, reference, dim='svd', comparison=None):
+def _crpss_es(forecast, reference, **kwargs):
+    """CRPSS Ensemble Spread.
+
+    .. math:: CRPSS = 1 - \\frac{CRPS(\\sigma^2_f)}{CRPS(\\sigma^2_o}))
+
+    References:
+        * Kadow, Christopher, Sebastian Illing, Oliver Kunst, Henning W. Rust,
+          Holger Pohlmann, Wolfgang A. Müller, and Ulrich Cubasch. “Evaluation
+          of Forecasts by Accuracy and Spread in the MiKlip Decadal Climate
+          Prediction System.” Meteorologische Zeitschrift, December 21, 2016,
+          631–43. https://doi.org/10/f9jrhw.
+
+    # TODO: not yet fully understood!
```

Comment:
Same question here. We should not implement any metrics that we aren't 100% are correct, since users might rely on them. You could drop any metrics from this PR that aren't 100% and then have an isolated PR in the future to focus on each of them individually.

Reply:
Users shouldn't blindly do this from my point of view: `users might rely on them`. 
`crpss_es` checks whether ensemble spread can be used as a reference for the forecast uncertainty.
`less` checks whether overdispersive or underdispersive. so its a sign check see Kadow 2016.

I didn't run these two on miklip prediction system output. I will.

Reply:
```python
hind = load_dataset('CESM-DP-SST').load()['SST']
reconstruction = load_dataset('FOSI-SST').load()['SST']

reconstruction = reconstruction-reconstruction.mean()
compute_hindcast(hind, reconstruction,metric='crpss_es', dim='member', comparison='m2r').plot(label='crpss_es')
compute_hindcast(hind, reconstruction,metric='less', comparison='m2r',dim='init').plot(label='less')
plt.legend()
```
![less](https://user-images.githubusercontent.com/12237157/65345004-6e835e00-dbd9-11e9-9964-5b415d04ba3b.png)

CRPSS_ES is negative as it should. perfect would be 0. Degrades so ensemble spread can be less and less used as forecast uncertainty. LESS now determines the sign: positive means that the ensemble is underdispersive.

I will also recheck with MPI output to verify the plots in that Kadow paper.